docs/features/routing.md

```diff
@@ -4,9 +4,24 @@ Permits routing records to specific topic using [`jmespath`](https://jmespath.or
 The routes are declared under `routes` with `jmesPath` as a matching condition and `topic` as the name for the topic to route the message to.
 
 If no route is matched, the record lands in `errorTopic`
+
+### Usage
+
+First, we need to declare the `topics` on the Kahpp instance.
+For example:
+```yaml
+kahpp:
+  topics:
+    source: snapshot.response
+    email: action.email
+    salesforce: action.salesforce
+    slack: action.slack
+    error: action-discovery.error
```

Comment:
The examples contain domain-specific references - I'd replace them with more generic ones.